GitUI/CommandsDialogs/SettingsDialog/CheckSettingsLogic.cs

```diff
@@ -178,7 +178,7 @@ bool TestGitCommand(string command)
             {
                 try
                 {
-                    string output = new Executable(command).GetOutput();
+                    string output = new Executable(command).GetOutput(throwOnErrorOutput: false);
```

Comment:
Avoid commands that fail when they do not have to
```suggestion
                    string output = new Executable(command).GetOutput(arguments: " --version");
```